packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -941,6 +941,8 @@ new lambda.Function(this, 'Function', {
 });
 ```
 
+See [Asset Bundling](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3_assets-readme.html#asset-bundling) for more details on bundling options
```

Comment:
I don't think this will actually translate correctly across the various ways we publish our documentation. I think we can just remove this here and only include the entry in the s3 assets package.